Dockerfile.wsc

```diff
@@ -0,0 +1,39 @@
+# Builder image
+FROM golang:windowsservercore-ltsc2022 as builder
+
+COPY . /neo-go
+
+WORKDIR /neo-go
+
+ARG REPO=repository
+ARG VERSION=dev
+
+SHELL ["cmd", "/S", "/C"]
+RUN go env -w CGO_ENABLED=0
+ENV GOGC=off
+
+RUN go build -trimpath -v -o ./bin/neo-go.exe -ldflags="-X %REPO%/pkg/config.Version=%VERSION%" ./cli/main.go
+
+# Executable image
+FROM mcr.microsoft.com/windows/servercore:ltsc2022
+
+ARG VERSION
+LABEL version=%VERSION%
+
+WORKDIR /
+
+COPY --from=builder /neo-go/config /config
+COPY --from=builder /neo-go/.docker/privnet-entrypoint.ps1 /usr/bin/privnet-entrypoint.ps1
+COPY --from=builder /neo-go/bin/neo-go.exe /usr/bin/neo-go.exe
+# TODO: Windows Server doesn't have default crt, need to export certs from certmgr
```

Comment:
I think we need to create a separate issue for that, because it's not that easy to release sertificates for windows.

Reply:
But do we need to do this on Windows? I'm not sure where the client gets certificates from there.

Reply:
I'm not sure also, so I think we can go without it until it is requested by users. I've removed this TODO.